metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<p>This appears extremely sudden. This should be already alluded to in the introduction. Now explain here: What are we doing, how are we doing this, why are we doing this, why is it scientifically relevant etc? Keep it short with 2-3 sentences total but provide more context!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>